client/src/modules/cohort/index.jsx

```diff
@@ -120,6 +120,11 @@ class Cohort extends PureComponent {
     return cohortDetails && cohortDetails.isArchived;
   };
 
+  checkIfAuthorized = () => {
```

Comment:
not sure is the name of this method is proper, 
cause you're checking if the user is admin and not if authorized.